crates/typst-layout/src/math/fragment.rs

```diff
@@ -390,20 +388,35 @@ impl GlyphFragment {
         frame
     }
 
-    pub fn make_scriptsize(&mut self, ctx: &MathContext) {
+    pub fn make_script_size(&mut self, ctx: &MathContext) {
         let alt_id =
-            script_alternatives(ctx, self.id).and_then(|alts| alts.alternates.get(0));
+            ctx.ssty_table.as_ref().and_then(|ssty| ssty.try_apply(self.id, None));
+        if let Some(alt_id) = alt_id {
+            self.set_id(ctx, alt_id);
+        }
+    }
 
+    pub fn make_script_script_size(&mut self, ctx: &MathContext) {
+        let alt_id = ctx.ssty_table.as_ref().and_then(|ssty| {
+            ssty.try_apply(self.id, Some(1))
```

Comment:
This `1` maybe deserves a constant or comment.

Reply:
I've added a comment explaining its meaning.